### PR TITLE
Only print skip message for disabled tests

### PR DESCRIPTION
**Pull request by fzipi**
Use File.Seek() to move to end of file for next read

### CODE REVIEW

File: waflog/waflog_test.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+	ll := NewFTWLogLines()
+	// Loop through each option
+	for _, opt := range []FTWLogOption{
+		WithStartMarker([]byte("#")),
+		WithEndMarker([]byte("#")),
+		WithLogFile("test"),
+	} {
+		// Call the option giving the instantiated
+		// *House as the argument
```
Comment (theseion) [inline waflog/waflog_test.go:22]: `*House`??

File: runner/run_test.go [hunk 2]
```diff
@@ -555,18 +553,9 @@ func TestCloudRun(t *testing.T) {
 				} else if stage.Output.NoLogContains != "" {
 					responseStatus = 405
 				}
-				server, dest, logFilePath := newTestServerForCloudTest(t, responseStatus, logText)
+				server, dest, _ := newTestServerForCloudTest(t, responseStatus, logText)
```
Comment (theseion) [inline runner/run_test.go:556]: Remove log file return value from `newTestServerForCloudTest`.

File: config/config_test.go [hunk 3]
```diff
@@ -159,10 +162,23 @@ func TestNewConfigFromStringHasDefaults(t *testing.T) {
...
+func TestNewConfigFromFileRunMode(t *testing.T) {
+	filename, _ := utils.CreateTempFileWithContent(yamlCloudConfig, "test-*.yaml")
+	defer os.Remove(filename)
+
+	if err := NewConfigFromFile(filename); err != nil {
+		t.Error(err)
+	}
+
+	if FTWConfig.RunMode != CloudRunMode {
+		t.Errorf("unexpected default value '%s' for run mode", FTWConfig.RunMode)
```
[the remark this answers is not in the document]
Comment (fzipi) [reply config/config_test.go:182]: Added.

File: runner/run.go [hunk 4]
```diff
@@ -58,6 +45,10 @@ func Run(include string, exclude string, showTime bool, output bool, ftwtests []
 
 	printSummary(output, runContext.Stats)
 
+	if err := logLines.Cleanup(); err != nil {
+		log.Fatal().Err(err).Msg("Failed to cleanup log file")
```
[the remark this answers is not in the document]
Comment (fzipi) [reply runner/run.go:49]: Moved to a new func and `defer`red

File: waflog/waflog.go [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+		logFile:     nil,
+		FileName:    config.FTWConfig.LogFile,
+		StartMarker: nil,
+		EndMarker:   nil,
+	}
+
+	// Loop through each option
+	for _, opt := range opts {
+		// Call the option giving the instantiated
+		// *House as the argument
```
[the remark this answers is not in the document]
Comment (fzipi) [reply waflog/waflog.go:22]: Haha, bad copy and paste :D

File: waflog/waflog.go [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+		return ll.logFile.Close()
+	}
+	return nil
+}
+
+func (ll *FTWLogLines) openLogFile() error {
+	// Using a log file is not required in cloud mode
+	if config.FTWConfig.RunMode == config.DefaultRunMode {
+		if ll.FileName != "" && ll.logFile == nil {
+			var err error
```
[the remark this answers is not in the document]
Comment (fzipi) [reply waflog/waflog.go:66]: See comment below on why.

File: waflog/waflog.go [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+			ll.logFile, err = os.Open(ll.FileName)
```
[the remark this answers is not in the document]
Comment (fzipi) [reply waflog/waflog.go:67]: This doesn't work as expected on structs. See https://github.com/golang/go/issues/6842. Try it :D

File: waflog/waflog_test.go (hunk 1 above)
Comment (fzipi) [reply waflog/waflog_test.go:22]: Bad copy :)

File: runner/run_test.go (hunk 2 above)
Comment (fzipi) [reply runner/run_test.go:556]: Done.

File: waflog/waflog.go (hunk 7 above)
[the remark this answers is not in the document]
Comment (theseion) [reply waflog/waflog.go:67]: Oh... I think a stumbled over this the last time too :)

**Review by theseion — APPROVED** [review]
LGTM

**Pull request merged**